### DISCUSSION TITLE
新人报道，请多指教！

### DISCUSSION THREAD

**Discussion by DiamondDiavolo** [share121/inter-knot#560] [其他]
今天好大功夫才注册好绳网账号，但是手机端的适配不太好啊……电脑端该怎么用呢？用哪个浏览器更好？还有求教头像怎么更换！
![thumbmmexport1722940080696](https://github.com/user-attachments/assets/6b809eb5-d76f-4c95-ab4d-ad7fe8ad53cb)

**Comment by lllliufeng** [comment]
安一个油猴插件（比如edge的篡改猴扩展），在下载网站跳出来的脚本就可以了

**Comment by lllliufeng** [comment]
换头像的话在个人主页（Your profile）点头像编辑（edit）就行了，就是网不好得等会儿
